contact_detection/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 cmake_minimum_required(VERSION 3.0.2)
```

Comment:
contact_detection package will be deleted in melodic_devel_utilsmsg_refactor branch